src/content/guides/workload-cluster-labelling/index.md

```diff
@@ -12,6 +12,10 @@ owner:
 
 # Labelling workload clusters
 
+{{< platform_support_table aws="ga=v10.0.0" azure="ga" >}}
```

Comment:
@giantswarm/team-celestial Can you tell me when cluster labelling became available on Azure? Was that with the introduction of Cluster API types, in v13.0.0?

Reply:
Yes, since the 13.0.0 betas

Reply:
Thanks!